assets/css/styles.scss

```diff
@@ -664,3 +664,59 @@ body {
     }
   }
 }
+
+// Outreach partner section
+.outreach-grayscale {
+	.row {
+		text-align: center;
+		margin-left: auto;
+		margin-right: auto;
+		margin-bottom: 20px;
+		&:after {
+			content: "";
+			display: table;
```

Comment:
I would prefer the use of flex instead of table if possible. But I am open to what you think about it.

Reply:
Table works for me cause it's easy to write the styling on.
Flex is definitely better but I don't really get it. If you can, please feel free to change the styling to flex.